react/components/SKUSelector/utils/index.ts

```diff
@@ -226,3 +226,17 @@ export function changeImageUrlSize(
 
   return `${normalizedImageUrl}${queryStringSeparator}width=${width}&height=${height}&aspect=true`
 }
+
+export const DEFAULT_BOTTOM_MARGIN = 7
+const MAX_POSSIBLE_MARGIN = 11
+const MIN_POSSIBLE_MARGIN = 0
```

Comment:
Just out of curiosity, why these values?

Reply:
Each one represent a [tachyons class](https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css#L374)